packages/playground/src/components/pull-request/utils.tsx

```diff
@@ -93,3 +93,19 @@ export const timeDistance = (date1 = 0, date2 = 0, onlyHighestDenomination = fal
     minutes ? minutes + 'm' : hours || days ? '0m' : ''
   } ${seconds}s`
 }
+const LOCALE = Intl.NumberFormat().resolvedOptions?.().locale || 'en-US'
+
+/**
+ * Format a timestamp to medium format date (i.e: Jan 1, 2021)
+ * @param timestamp Timestamp
+ * @param dateStyle Optional DateTimeFormat's `dateStyle` option.
+ */
+export function formatDate(timestamp: number | string, dateStyle = 'medium'): string {
```

Comment:
Might be useful in other places as well, move it later on in a separate/common util.

Reply:
moved to outside utils file